optuna/samplers/_cmaes.py

```diff
@@ -131,10 +154,20 @@ def __init__(
         self._cma_rng = np.random.RandomState(seed)
         self._search_space = optuna.samplers.IntersectionSearchSpace()
         self._consider_pruned_trials = consider_pruned_trials
+        self._restart_strategy = restart_strategy
+        self._inc_popsize = inc_popsize
 
         if self._consider_pruned_trials:
             self._raise_experimental_warning_for_consider_pruned_trials()
 
+        # TODO(c-bata): Support BIPOP-CMA-ES.
+        if restart_strategy not in ("ipop", None,):
+            raise ValueError(
+                "restart_strategy={} is unsupported. Please specify: 'ipop' or None.".format(
+                    restart_strategy
+                )
+            )
+
```

Comment:
We can use the experimental decorator for the argument as follows.
```suggestion

@experimental("2.1.0", name="`restart_strategy != None` in CmaEsSampler")
    def _raise_experimental_warning_for_restart_strategy(self) -> None:
        pass

```